newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -31,14 +38,61 @@ const writeJSONFile = (object: Object, filepath: string): Promise<void> => {
   }
 };
 
+const writeProjectFiles = (
+  project: gdProject,
+  newFilepath: string,
```

Comment:
nitpicking on variable naming: in this function, the file path and the project path are actually not "new" - well they can be but we don't real care. We just want to save a project in a given folder with a given filepath. So you can replace:

newFilepath => filePath (capital P, even if I did not followed this convention in the rest of the file)
newProjectFolder => projectPath (to stay consistent with the xxxPath naming of other variables).